client/src/pages/header-main-test-page.js

```diff
@@ -0,0 +1,46 @@
+import Component from '@/components/component';
+
+import HeaderMain from '@/components/header/header-main.js';
+
+export default class TestPage extends Component {
+  constructor(props) {
+    super(props);
+
+    this.$dom = this.createDom('div', { className: 'test-page-wrapper' });
+
+    this.HeaderMain = new HeaderMain({
+      headerType: 'main',
+      placeName: '장소',
```

Comment:
```suggestion
      location: '장소',
```

HeaderMain 컴포넌트는 `headerType`를 사용하지 않고 있습니다.